--- origsrc/xpyb-1.0/src/Makefile.am	2008-06-03 22:06:08.000000000 -0500
+++ src/xpyb-1.0/src/Makefile.am	2008-10-26 02:07:51.796032100 -0500
@@ -12,7 +12,7 @@
 xcb_la_LIBADD = $(LIBXCB_LIBS)
 xcb_la_CPPFLAGS = -I$(PYTHON_INCLUDE)
 xcb_la_CFLAGS = -g $(CWARNFLAGS)
-xcb_la_LDFLAGS = -module
+xcb_la_LDFLAGS = -module -avoid-version -no-undefined -lpython$(PYTHON_VERSION)
 xcb_la_SOURCES = conn.c constant.c cookie.c error.c event.c except.c \
 		 ext.c extkey.c iter.c list.c module.c protobj.c \
 		 reply.c request.c response.c struct.c union.c void.c \
